package.json

```diff
@@ -30,10 +30,13 @@
   "homepage": "https://github.com/j-rewerts/IFTTT-Edmonton#readme",
   "dependencies": {
     "body-parser": "^1.18.3",
+    "colornames": "^1.1.1",
     "express": "^4.16.4",
+    "lodash": "^4.17.11",
```

Comment:
So the current uses of this are for capitalizing and lowercasing words it looks like. We can do that using regular old JS.